salt/_logging/impl.py

```diff
@@ -1,17 +1,31 @@
 # -*- coding: utf-8 -*-
-'''
+"""
     salt._logging.impl
     ~~~~~~~~~~~~~~~~~~
 
     Salt's logging implementation classes/functionality
-'''
+"""
 
 # Import python libs
 from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
 import re
 import sys
 import types
-import logging
+
+# Import 3rd-party libs
+import salt.ext.six as six
+
+# from salt._logging.handlers import SysLogHandler
+# from salt._logging.handlers import RotatingFileHandler
+# from salt._logging.handlers import WatchedFileHandler
+# Import Salt libs
+from salt._logging.handlers import StreamHandler, TemporaryLoggingHandler
+from salt._logging.mixins import LoggingMixinMeta, NewStyleClassMixin
+from salt.exceptions import LoggingRuntimeError
+from salt.utils.ctx import RequestContext
+from salt.utils.textformat import TextFormat
```

Comment:
These imports need to go back to where they were. See the comment above where we define the new log levels.